lib/fog/azurerm/models/network/subnet.rb

```diff
@@ -21,15 +21,15 @@ def self.parse(subnet)
           hash['address_prefix'] = subnet['properties']['addressPrefix']
           hash['network_security_group_id'] = subnet['properties']['networkSecurityGroup']['id'] unless subnet['properties']['networkSecurityGroup'].nil?
           hash['route_table_id'] = subnet['properties']['routeTable']['id'] unless subnet['properties']['routeTable'].nil?
-          hash['ip_configurations_ids'] = subnet['properties']['ipConfigurations'].map { |item| item['id'] } unless subnet['properties']['ipConfigurations'].nil?
+          hash['ip_configurations_ids'] = subnet['properties']['ipConfigurations']
```

Comment:
Since ip_configuration_ids is a list, it should have ip_config_ids in following format [id1, id2, id3, ...]